docs/src/1_Mechanics/07_Items.md

```diff
@@ -0,0 +1,113 @@
+<!-- markdownlint-disable MD013 MD024 -->
+<!-- DEVELOPERS: Please edit corresponding yaml -->
+
+# Armor
+
+**Light Armor**
+
+- Cost: 5 gp
+- Description: Replace AR with 3.
```

Comment:
We'll probably want to add `Item Type` as the first field for each item in the .md, i.e. Armor, Weapon, Tool, Consumable, etc.